quiffen/core/qif.py

```diff
@@ -109,6 +109,7 @@ def parse(
         separator: str = "\n",
         day_first: bool = False,
         encoding: str = "utf-8",
+        progress=None,
```

Comment:
What type is `progress` here? What's it supposed to represent? It's not immediately clear from the naming.